### DISCUSSION TITLE
Improve first startup experience

### DISCUSSION THREAD

**Discussion by samuel-w** [borgbase/vorta#832] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently on first startup we just dump the user into the UI, with builtin no explanation on how to use it. 
There is a guide on vorta.borgbase.com, but I would like for it to be an interactive setup and self-contained in Vorta. 

**Describe the solution you'd like**
On first setup, guide the user through:

1. Restoring a backed up profile, and skip the rest if that happens (once #462 gets resolved)
2. Adding a repository
3. Adding sources
4. Scheduling backups
5. Creating a backup of the new profile (once #462 gets resolved)

And add a misc option to run this sequence every time a new profile is added

**Describe alternatives you've considered**
Improve the guide on vorta.borgbase.com with points 4 and 5.

**Comment by sudwhiwdh** [comment]
Could this be usefully linked with https://github.com/borgbase/vorta/issues/317?

**Comment by ghost** [comment]
for reference and inspiration #317 (comment)
